packages/engine/Source/Scene/Scene.js

```diff
@@ -1998,18 +2005,31 @@ Scene.prototype.updateFrameState = function () {
 };
 
 /**
+ * Check whether a draw command will render anything visible in the current Scene,
+ * based on its bounding volume.
+ *
+ * @param {DrawCommand} [command] The draw command
+ * @param {CullingVolume} cullingVolume The culling volume of the current Scene.
```

Comment:
The order of "optional, required, optional" parameters is non-ideal. Again, not strictly required for this PR, but it would be best if we ordered the signature with 'required, optional, optional".